portfolio-project-react/src/constants.jsx

```diff
@@ -0,0 +1,3 @@
+export const LABEL_START_DATE = "Start Date:";
+export const LABEL_END_DATE = "End Date:";
+export const LABEL_CURR_EMP = "Current Employer";
```

Comment:
CURR_EMP could also mean current employee, so when using this contant in other files, it might give un-expected results so better to use fullnames